### ISSUE TITLE
Carousel - Pause button losing focus on activation

### ISSUE THREAD

**Issue by mcferrenm** [eBay/ebayui-core#1060]
# Bug Report

## Description

ACTION PERFORMED
1. Using Chrome Version 78.0.3904.87
2. Tab to focus 'Pause' button in the main carousel and press Enter to activate
3. Press Tab and check focus location

EXPECTED RESULT
When 'Pause' button in the main carousel is activated focus stays on this button (which becomes 'Play').

ACTUAL RESULT
When 'Pause' button in the main carousel is activated focus is lost and in Chrome activeFocus moves to body.

**Comment by ianmcburnie** [comment]
Only latest 4.x and 5.x versions are supported. So any fix would go in 5.0.x and 4.5.x.

**Comment by agliga** [comment]
This seems to be related to the fact that we have an if statement around the button so it creates a whole new element when clicked. 
I added a way to fix it by adding a key and added a focus. But will investigate if we can change the rerender.

**Comment by agliga** [comment]
Actually, there is an underlying problem with this which I noticed when I did the change for this.
It seems like when the carousel has focus the autoplay does not run. It stays on the same page until focus is removed. If you toggle the play/pause button it will slide once but it will not slide until the focus is removed.
So in a sense, not having focus on play/pause makes sense since it blocks the automove. 
One thing we can probably do is to

**Comment by ianmcburnie** [comment]
The carousel is intentionally put into a paused state when any interactive carousel element has focus (and hover too, I believe). This is so that the carousel doesn't move when a user is trying to interact with an item.

I think the solution to the issue is just to call .`focus()` again on the button.

**Comment by agliga** [comment]
Published `@ebay/ebayui-core@4.5.10` which should have this fix
